src/redux/books/bookSlice.js

```diff
@@ -0,0 +1,107 @@
+import { createSlice, createAsyncThunk } from '@reduxjs/toolkit';
+import axios from 'axios';
+
+const url = 'https://us-central1-bookstore-api-e63c8.cloudfunctions.net/bookstoreApi/apps/K3rr9q3I2WcXGF3EcDhn/books';
+
+const initialState = {
+  books: [],
+  isLoading: false,
+  error: false,
+  success: false,
+};
+
+const fetchBooks = createAsyncThunk('books/fetchBooks', async () => {
+  try {
+    const res = await axios.get(url);
+    return res.data;
+  } catch (error) {
+    return error;
+  }
+});
+
+const removeBook = createAsyncThunk('books/removeBook', async (id) => {
+  try {
+    const res = await axios.delete(`${url}/${id}`);
+    return res;
+  } catch (error) {
+    return error;
+  }
+});
```

Comment:
- [ ] Although you did an excellent job of utilizing createAsyncThunk to retrieve data from the API, there appears to be an issue with your application when a user attempts to delete a book after adding multiple books. The books cannot be removed, which needs to be addressed.


https://user-images.githubusercontent.com/105117832/231622720-02dc2086-5612-408e-8d41-6eada57aab19.mp4

Reply:
checking same said issue after refreshing it does  not show any book after deleting them